api/desecapi/tests/base.py

```diff
@@ -609,6 +611,62 @@ def request_pdns_update_catalog(cls):
             "priority": 1,  # avoid collision with DELETE zones/(?P<id>[^/]+)$ (httpretty does not match the method)
         }
 
+    def request_pch_zone_create(self, name, **kwargs):
+        def request_callback(r, _, response_headers):
+            try:
+                self.assertEqual(
+                    r.parsed_body,
+                    {"zones": [name]},
+                    f"Expected PCH zone creation request for {name}, but got '{r.parsed_body}'.",
+                )
+            finally:
+                return [
+                    201,
+                    response_headers,
+                    json.dumps(
+                        {
+                            "status": True,
+                            "message": "Zone(s) ADDED",
+                            "zones": [name],
+                        }
+                    ),
+                ]
+
+        return {
+            "method": "POST",
+            "uri": re.compile("^" + settings.PCH_API + self.PCH_ZONE_CREATE),
+            "body": request_callback,
+            **kwargs,
+        }
+
+    def request_pch_zone_delete(self, name, **kwargs):
+        def request_callback(r, _, response_headers):
+            try:
+                self.assertEqual(
+                    r.parsed_body,
+                    {"zones": [name]},
+                    f"Expected PCH zone deletion request for {name}, but got '{r.parsed_body}'.",
+                )
+            finally:
+                return [
+                    200,
+                    response_headers,
+                    json.dumps(
+                        {
+                            "status": True,
+                            "message": "some message",
```

Comment:
The message is `"Zone(s) deleted`"` (probably doesn't matter)

Reply:
adjusted